telemetry/sql/000013_peer_conn_failure.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS peerConnFailure (
+    id SERIAL PRIMARY KEY,
+    createdAt INTEGER NOT NULL,
+    peerId VARCHAR(255) NOT NULL,
+    nodeName VARCHAR(255) NOT NULL,
+    nodeKeyUid VARCHAR(255) NOT NULL,
+    timestamp INTEGER NOT NULL,
+    statusVersion VARCHAR(31),
+    failureCount INTEGER NOT NULL,
+    failedPeerId VARCHAR(255) NOT NULL,
+    CONSTRAINT peerConnFailure_unique unique(timestamp, peerId, failedPeerId, failureCount)
```

Comment:
I am not sure about this index - the chance that the same peer will report the same failing peer with the same failure count is slim, but definitely not non-existent - any particular reason for this constraint?

Reply:
Including the timestamp should make it unique enough imo. It's mostly to prevent obvious duplicates (like if for whatever reason the client reports the same exact data twice).

Reply:
Huh, sorry, I missed the timestamp in there 🤦‍♂️